src/popup/pages/Send/index.js

```diff
@@ -399,16 +399,9 @@ class SendPage extends React.Component {
   setBtnStatus = () => {
     const {toAddressCanInputDefaultValue} = this.pageConfig
     let toAddress = this.state.toAddress || toAddressCanInputDefaultValue
-    if (toAddress.length > 0
-      && this.state.amount.length > 0) {
-      this.callSetState({
-        btnClick: true
-      })
-    } else {
-      this.callSetState({
-        btnClick: false
-      })
-    }
+    this.callSetState({
+      btnClick: toAddress.length > 0 && this.state.amount.length > 0
```

Comment:
the existence of setBtnStatus and btnClick in the state at all seems like a weird departure from reactive UI programming. and then there's code in various places that are careful to call setBtnStatus whenever the inputs change. it seems to go through so much trouble to reimplement reactivity

Reply:
Ah yeah, removed.